### ISSUE TITLE
'MicroExpNet' object has no attribute 'maxpool2d'

### ISSUE THREAD

**Issue by LetFoxRun** [cuguilke/microexpnet#10]
I guess you have forgotten the function "maxpool2d" while you update last time. I try to write it myself.However, when I add:
`def maxpool2d(self,x):
return tf.nn.max_pool(x, ksize=[1, 2, 2, 1], strides=[1, 2, 2, 1], padding='VALID') `
I got the error " Assign requires shapes of both tensors to match. lhs shape= [1152,48] rhs shape= [3872,16]".
So could you please add the code or new model file to fix this bug?
Many thanks!

**Comment by cuguilke** [comment]
Yes, you are right. I've uploaded the fixed code. Correct pretrained models will be added too. Until then, you can just comment out the weight deployer, that will solve the shape matching error.
Thanks for letting me know.
